ui/appui-react/src/appui-react/syncui/SyncUiEventDispatcher.ts

```diff
@@ -77,8 +77,6 @@ export class SyncUiEventDispatcher {
   private static _connectionUnregisterFuncs = new Array<() => void>();
   private static _iModelConnection?: IModelConnection;
 
-  /** @internal - used for testing only */
```

Comment:
I guess the idea is not to have `internal` methods in `public` classes? But if this method is only intended for testing, is it ok to make it non-internal?

Reply:
Not really, you can have an internal method in a public class, however, one of our consumers is using this method, so I've decided to simply remove the internal tag and promote this method to public.